content.md

```diff
@@ -1423,7 +1426,9 @@ succeeds; the cost is not paid if the action is blocked. The costs of
 action modifiers and reaction cards are always paid when the cards are
 played, regardless of the success of the action.
 
-
+> **ADVANCED RULES**\
+> If an effect **ends the action** after block resoltion, that makes the action end unsuccessfully.
```

Comment:
```suggestion
> If an effect **ends the action** after block resolution, that makes the action end unsuccessfully.
```

but also afaik "after block resolution" is redundant or event incorrect, these effects generally happen *before* block resolution

Reply:
Thanks for the review :) I tried to include the ruling below. Is there any way I could word it better?

c) Failing action vs ended action
For less disparity, some cards that were causing an action to end unsuccessfully before resolution now make it fail; _if the action had resolved (e.g., it had been blocked), those cards make them end unsuccessfully._ That way, the action has resolved in both cases.
    Actions that are ended are over: action modifiers and reactions cannot be played anymore.
    Actions that fail are similar to actions that have been blocked (notice that a blocked action fails) except that there is no blocker and no combat. Action modifiers and reaction can still be played, if their text does not refer to a blocked action (e.g., Freak Drive cannot be played at superior after a failed action since it was not blocked, and cannot be played at basic since the action was not successful)